CourseSketchProjects/CourseSketchDatabaseServer/src/main/java/database/institution/mongo/BankProblemManager.java

```diff
@@ -80,7 +83,11 @@ public static String mongoInsertBankProblem(final DB dbs, final SrlBankProblem p
                 .append(SOURCE, problem.getSource())
                 .append(QUESTION_TYPE, problem.getQuestionType().getNumber())
                 .append(SCRIPT, problem.getScript())
-                .append(KEYWORDS, problem.getOtherKeywordsList());
+                .append(KEYWORDS, problem.getOtherKeywordsList())
+                        // FUTURE: fix access issues for bank problems
```

Comment:
I do believe this is saying that we need to work out how instructors limit access to bank problems with a registration key